.travis.yml

```diff
@@ -1,24 +1,52 @@
 language: node_js
 node_js:
-  - "8"
+- '8'
 cache:
   directories:
-    - node_modules
+  - node_modules
 install: npm install
 before_script: npm run dist
 script: npm test
-before_deploy: "echo 'All unit tests passed; Preparing to deploy Staff Picks to AWS'"
+before_deploy: echo 'All unit tests passed; Successfully built distribution assets;
+  Preparing to deploy Staff Picks to AWS'
 deploy:
+- provider: elasticbeanstalk
   skip_cleanup: true
-  provider: elasticbeanstalk
-  access_key_id: $AWS_PRODUCTION_ACCESS_KEY_ID
-  secret_access_key: "$AWS_PRODUCTION_ACCESS_SECRET"
+  access_key_id: "$aws_access_key_id_production"
+  secret_access_key: "$aws_secret_access_key_production"
   region: us-east-1
-  app: $AWS_APP_NAME
-  env: $AWS_ENVIRONMENT_NAME
+  app: "$aws_app_name"
+  env: "$aws_environment_name_production"
   bucket_name: elasticbeanstalk-us-east-1-946183545209
-  bucket_path: $AWS_APP_NAME
+  bucket_path: "$aws_app_name"
   on:
     repo: NYPL/staff-picks
     branch: master
-after_deploy: "echo 'Successfully executed deploy trigger for Staff Picks on AWS'"
+- provider: elasticbeanstalk
+  skip_cleanup: true
+  access_key_id: "$aws_access_key_id_production"
+  secret_access_key: "$aws_secret_access_key_production"
+  region: us-east-1
+  app: "$aws_app_name"
+  env: "$aws_environment_name_qa"
+  bucket_name: elasticbeanstalk-us-east-1-946183545209
+  bucket_path: "$aws_app_name"
+  on:
+    repo: NYPL/staff-picks
+    branch: qa
+- provider: elasticbeanstalk
+  skip_cleanup: true
+  access_key_id: "$aws_access_key_id_development"
+  secret_access_key: "$aws_secret_access_key_development"
+  region: us-east-1
+  app: "$aws_app_name"
+  env: "$aws_environment_name_development"
+  bucket_name: elasticbeanstalk-us-east-1-224280085904
+  bucket_path: "$aws_app_name"
+  on:
```

Comment:
You have this above with `branch: master`. Is it okay to two of these??

Reply:
Since QA and Prod environments both live in digital-dev account, they use the same credentials.